int16 io_port
int16 value
---
int16 status
